.circleci/config.yml

```diff
@@ -75,8 +75,9 @@ jobs:
       - run:
           name: DangerCI
           command: npm run danger
+      - run:
+          name: Bundle size analysis
+          command: npx -p iltorb bundlesize --debug
```

Comment:
can we get this command in a package.json script? i'll be porting over functionality to a different CI platform in a few weeks and it is also better practice to have these commands in a reusable place, rather than straight in the config.

Reply:
Sure.